packages/core/src/secret/__tests__/secret-aes256.test.ts

```
@@ -255,31 +329,72 @@
+        key: 'test-key',
+      });
+      expect(console.warn).toHaveBeenCalledWith(
+        'decodeSensitiveText() 已弃用 (deprecated). Please use decodeSensitiveTextAsync() instead',
+      );
+    });
+    beforeEach(() => {
+      jest.spyOn(console, 'warn').mockImplementation(() => {});
+      jest.spyOn(console, 'trace').mockImplementation(() => {});
+    });
+
+    afterEach(() => {
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)